### PR TITLE
Add Fdplast and Bestfilament material

### PR DESCRIPTION
**Pull request by kimer2002**
These are plastic manufacturers from Russia. These profiles are used in ZAV 3D printers
I add ZAV 3D printer profiles to Cura and these materials will be used.
https://github.com/Ultimaker/Cura/pull/8501

### CODE REVIEW

File: fdplast_abs_tomato.xml.fdm_material [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>FDplast</brand>
+      <material>ABS</material>
+      <color>Tomato</color>
+      <label>FD ABS Tomato</label>
+    </name>
+    <adhesion_info>??????????? ???????????? ???? ? ?????????? ???????? ???????.</adhesion_info>
```
Comment (kad) [inline fdplast_abs_tomato.xml.fdm_material:10]: non-UTF8 charset?

**Review by kad — COMMENTED** [review]
additional nit: EOLs at end of the files. Some are missing, some have more than one.

**Comment by kimer2002** [comment]
Сorrected

File: bestfilament_abs_skyblue.xml.fdm_material [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>Best Filament</brand>
+      <material>ABS</material>
+      <color>SkyBlue</color>
+      <label>BF ABS SkyBlue</label>
+    </name>
+    <adhesion_info>Обязательно использовать клей и желательно закрытый принтер.</adhesion_info>
```
Comment (Ghostkeeper) [inline bestfilament_abs_skyblue.xml.fdm_material:10]: Our profiles are not translated. Could you write the adhesion info and description in English? That captures most of Cura's user base.

**Review by Ghostkeeper — DISMISSED** [review]

File: bestfilament_abs_skyblue.xml.fdm_material (hunk 2 above)
Comment (kimer2002) [reply bestfilament_abs_skyblue.xml.fdm_material]: Translated into english

**Review by rburema — APPROVED** [review]
Everything seems ok now.

**Pull request merged**